units/systemd-networkd-persistent-storage.service

```diff
@@ -0,0 +1,27 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Notify if the persistent storage for systemd-networkd.service is ready
```

Comment:
Descriptions must be "Title Case" and work well with "Starting …" and "Started …".
"Notify systemd-networkd.service that Persistent Storage is Available" ?
Or maybe even "Enable Persistent Storage in systemd-networkd"?